source/Model.cpp

```diff
@@ -49,6 +50,25 @@ namespace GTA
 	}
 	bool Model::IsVehicle::get()
 	{
-		return IsBicycle || IsBike || IsBoat || IsCar || IsHelicopter || IsPlane || IsQuadbike || IsTrain;
+		return IsBicycle || IsBike || IsBoat || IsCar || IsHelicopter || IsPlane || IsQuadbike || IsTrain; //TODO Exchange with IS_MODEL_A_VEHICLE?
+	}
+	bool Model::IsInCdImage::get()
+	{
+		return Native::Function::Call<bool>(Native::Hash::IS_MODEL_IN_CDIMAGE, this->mHash);
+	}
+
+	void Model::Request()
```

Comment:
Instead of splitting Request and BlockTillLoaded into two functions, how about providing a "void Model::Request(bool blockUntilLoaded)" overload?